### DISCUSSION TITLE
Collecting frictions of the on boarding process

### DISCUSSION THREAD

**Discussion by AltiMario** [e-Learning-DAO/POCRE#216] [Ideas]
This discussion is about what you think can be improved in the experience of the user who lands on the platform for the first time.
Please give your thoughts on this list of problems:
1. I don't know what is a wallet when I click on the login button. **Do we need a better page/popup with a further description?**
2. if I'm not logged in, I don't see anything in the "creations" and "recognitions" (search for a wallet is a planned feature). **Do we need to show content to not logged users?**

**Comment by huzaifa-99** [comment]
Nice initiative. 
1. That's needed in my opinion, because its not very clear for first time users. Something like an about page or even description on the login page/model might be good
2. I think for now we could show some form of creations on the home page of creations for non-auth users (these could be latest/trending creation for example). For recognitions we could block/hide that from the menu for non-auth users
